go-runtime-mixin/dashboards/go-runtime.json

```diff
@@ -0,0 +1,1152 @@
+{
```

Comment:
I'd consider adding a table at the top of the dashboard, broken down by job & instance, showing things like the go version and allowing drill down to a specific job / instance.

Welcome to punt this to later though, just please open a ticket for it.

Reply:
Certainly seems useful, this dash does feel kinda naked without such a thing. Here, my relative inexperience with Grafana bites me - is there an example of what you have in mind out there that i could work from?

(Aside, seems like `client_golang` should also probably emit a new gauge with `runtime.BuildInfo.Path` and `runtime.BuildInfo.Module.Version`)

Reply:
Yeah, checkout the tables in the kubernetes mixin: https://admin-ops-tools1.grafana.net/grafana/d/efa86fd1d0c121a26444b636a3f509a8/kubernetes-compute-resources-cluster?orgId=1&refresh=10s

Reply:
OK, gonna slate that for a follow-up.